### ISSUE TITLE
FileHistory view with follow renames:on shows merge commits instead of individual commits

### ISSUE THREAD

**Issue by vnkavali** [gitkraken/vscode-gitlens#1560]
File history view shows merge-commits when follow-renames is ON, but shows the commits instead of merge commits when follow-renames is OFF.

for the following git graph
```
*   b706caa (HEAD -> master) Merge branch 'test'
|\  
| * 52aa22a (test) unswap
* | ee0423c Add file3
|/  
* 01e3428 Swap file1 and file2
* 77c154c Add src/file2
* 504830f add file1
```

Expected behaviour:
```
$git log --follow --oneline -- file1.txt
52aa22a unswap
01e3428 Swap file1 and file2
77c154c Add src/file2
504830f add file1
```

Gitlens FileHistory with follow-renames:on
![Screenshot from 2021-06-29 15-09-53](https://user-images.githubusercontent.com/58797059/123746095-23d56080-d8ec-11eb-8ddd-eacba2df5dd1.png)

Is there anyway I can show the individual commits (`52aa22a unswap`) instead of the merge-commit?

**Comment by JonParton** [comment]
I have the same issue when renaming a file within a branch that has not been merged!

**Comment by eamodio** [comment]
Duplicate of https://github.com/gitkraken/vscode-gitlens/issues/2944 which has shipped in GL 14.4
